widget/ui/src/components/Chip/Chip.styles.ts

```diff
@@ -9,12 +9,16 @@ export const ChipContainer = styled('button', {
   cursor: 'pointer',
   transition: 'all 0.35s',
   backgroundColor: '$neutral200',
-  border: 'none',
+  border: '1px solid transparent',
 
   '&:hover': {
     backgroundColor: '$surface600',
   },
 
+  '&:focus': {
+    outline: '1px solid gray',
```

Comment:
do we have gray in our theme?

Reply:
I didn't find anything in figma, I put this temporarily